+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Mon Dec  5 08:46:51 2022 - INFO  - Start environmental preparation.
+ DNF_INSTALL sanlk-reset
+ pkgs=sanlk-reset
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs sanlk-reset --node 1 --tempfile ''
Mon Dec  5 08:47:10 2022 - ERROR - pkgs:(sanlk-reset) not found
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ service=sanlk-resetd.service
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Mon Dec  5 08:47:11 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Mon Dec  5 08:47:12 2022 - INFO  - Start testing...
+ test_execution sanlk-resetd.service
+ service=sanlk-resetd.service
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-05 08:47:12'
+ test_restart sanlk-resetd.service
+ service=sanlk-resetd.service
+ systemctl restart sanlk-resetd.service
Failed to restart sanlk-resetd.service: Unit sanlk-resetd.service not found.
+ CHECK_RESULT 5 0 0 'sanlk-resetd.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='sanlk-resetd.service restart failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'sanlk-resetd.service restart failed'
+ LOG_ERROR 'sanlk-resetd.service restart failed'
+ message='sanlk-resetd.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'sanlk-resetd.service restart failed'
Mon Dec  5 08:47:13 2022 - ERROR - sanlk-resetd.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Mon Dec  5 08:47:14 2022 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status sanlk-resetd.service
+ grep 'Active: active'
Unit sanlk-resetd.service could not be found.
+ CHECK_RESULT 1 0 0 'sanlk-resetd.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='sanlk-resetd.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'sanlk-resetd.service restart failed'
+ LOG_ERROR 'sanlk-resetd.service restart failed'
+ message='sanlk-resetd.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'sanlk-resetd.service restart failed'
Mon Dec  5 08:47:20 2022 - ERROR - sanlk-resetd.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Mon Dec  5 08:47:21 2022 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop sanlk-resetd.service
Failed to stop sanlk-resetd.service: Unit sanlk-resetd.service not loaded.
+ CHECK_RESULT 5 0 0 'sanlk-resetd.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='sanlk-resetd.service stop failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'sanlk-resetd.service stop failed'
+ LOG_ERROR 'sanlk-resetd.service stop failed'
+ message='sanlk-resetd.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'sanlk-resetd.service stop failed'
Mon Dec  5 08:47:22 2022 - ERROR - sanlk-resetd.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Mon Dec  5 08:47:23 2022 - ERROR - ../common/common_lib.sh line 39
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status sanlk-resetd.service
+ grep 'Active: inactive'
Unit sanlk-resetd.service could not be found.
+ CHECK_RESULT 1 0 0 'sanlk-resetd.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='sanlk-resetd.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'sanlk-resetd.service stop failed'
+ LOG_ERROR 'sanlk-resetd.service stop failed'
+ message='sanlk-resetd.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'sanlk-resetd.service stop failed'
Mon Dec  5 08:47:30 2022 - ERROR - sanlk-resetd.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Mon Dec  5 08:47:31 2022 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start sanlk-resetd.service
Failed to start sanlk-resetd.service: Unit sanlk-resetd.service not found.
+ CHECK_RESULT 5 0 0 'sanlk-resetd.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='sanlk-resetd.service start failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'sanlk-resetd.service start failed'
+ LOG_ERROR 'sanlk-resetd.service start failed'
+ message='sanlk-resetd.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'sanlk-resetd.service start failed'
Mon Dec  5 08:47:32 2022 - ERROR - sanlk-resetd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Mon Dec  5 08:47:33 2022 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status sanlk-resetd.service
+ grep 'Active: active'
Unit sanlk-resetd.service could not be found.
+ CHECK_RESULT 1 0 0 'sanlk-resetd.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='sanlk-resetd.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'sanlk-resetd.service start failed'
+ LOG_ERROR 'sanlk-resetd.service start failed'
+ message='sanlk-resetd.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'sanlk-resetd.service start failed'
Mon Dec  5 08:47:40 2022 - ERROR - sanlk-resetd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Mon Dec  5 08:47:40 2022 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled sanlk-resetd.service
+ service=sanlk-resetd.service
++ systemctl is-enabled sanlk-resetd.service
Failed to get unit file state for sanlk-resetd.service: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Mon Dec  5 08:47:41 2022 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2022-12-05 08:47:12' -u sanlk-resetd.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of sanlk-resetd.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of sanlk-resetd.service'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload sanlk-resetd.service
+ service=sanlk-resetd.service
+ systemctl start sanlk-resetd.service
Failed to start sanlk-resetd.service: Unit sanlk-resetd.service not found.
+ systemctl reload sanlk-resetd.service
+ grep 'Job type reload is not applicable'
+ CHECK_RESULT 1 0 0 'Job type reload is not applicable for unit sanlk-resetd.service'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit sanlk-resetd.service'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Job type reload is not applicable for unit sanlk-resetd.service'
+ LOG_ERROR 'Job type reload is not applicable for unit sanlk-resetd.service'
+ message='Job type reload is not applicable for unit sanlk-resetd.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Job type reload is not applicable for unit sanlk-resetd.service'
Mon Dec  5 08:47:42 2022 - ERROR - Job type reload is not applicable for unit sanlk-resetd.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 91'
+ message='../common/common_lib.sh line 91'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 91'
Mon Dec  5 08:47:43 2022 - ERROR - ../common/common_lib.sh line 91
+ return 0
+ systemctl status sanlk-resetd.service
+ grep 'Active: active'
Unit sanlk-resetd.service could not be found.
+ systemctl status sanlk-resetd.service
+ grep 'inactive (dead)'
Unit sanlk-resetd.service could not be found.
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Mon Dec  5 08:47:44 2022 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Mon Dec  5 08:47:45 2022 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Mon Dec  5 08:47:46 2022 - INFO  - start environment cleanup.
+ systemctl stop sanlk-resetd.service
Failed to stop sanlk-resetd.service: Unit sanlk-resetd.service not loaded.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Mon Dec  5 08:47:49 2022 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Mon Dec  5 08:47:50 2022 - INFO  - Finish environment cleanup!
